spec/services/group_lister_spec.rb

```diff
@@ -28,8 +28,8 @@
     end
   end
 
-  context 'a node with a hierarchy' do
-    subject { described_class.new.list.sort_by(&:id).last }
+  context 'with a node with a hierarchy' do
+    subject { described_class.new.list.max_by(&:id) }
```

Comment:
nice, stealing this! Never seen it before...  :)

Reply:
I think this was rubocop - I can't claim to remember changing this manually!